.env.example

```
@@ -3,29 +3,33 @@
 
 # Core API Configuration (Required)
 # ---------------------------------
-UNRAID_URL=http://your-unraid-server
+UNRAID_API_URL=http://your-unraid-server
 UNRAID_API_KEY=your_api_key
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The UNRAID_API_KEY key should go before the UNRAID_API_URL key

(UnorderedKey)